curriculum/challenges/english/25-front-end-development/lecture-working-with-links/6716830dbaf95da9564f2e3b.md

```diff
@@ -10,6 +10,30 @@ dashedName: what-is-the-difference-between-slashes-a-single-dot-and-double-dot-i
 
 Watch the video and answer the questions below.
 
+# --transcript--
```

Comment:
this is the missing the title before the script

Reply:
I didn't include it here because it isn't spoken in [the video](https://www.freecodecamp.org/learn/full-stack-developer/lecture-working-with-links/what-is-the-difference-between-slashes-a-single-dot-and-double-dot-in-path-syntax). Should I add it anyway?

Reply:
oh ok, then that is fine to leave it as is